### PR TITLE
(blog) introducing nohoist 

### PR DESCRIPTION
**Pull request by connectdotz**
per @bestander [suggestion](https://github.com/yarnpkg/yarn/pull/4979?_pjax=%23js-repo-pjax-container#issuecomment-361076078) regarding [#4979](https://github.com/yarnpkg/yarn/pull/4979).

Didn't see much info regarding the blog format or submission process, hopefully, this is the way to do it, otherwise just let me know.

### CODE REVIEW

**Comment by Haroenv** [comment]
Deploy preview for *yarnpkg* ready!

Built with commit 251cdefc073401c05e9e2b879ac3f1ac7513b9bf

https://deploy-preview-777--yarnpkg.netlify.com

**Comment by Haroenv** [comment]
@yarnpkg/core

File: _posts/2018-02-15-nohoist.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+Since most 3rd-party libraries already worked in standalone projects, the ability to simulate such environment within workspaces should be able to unblock many known compatibility issues.
+
+### A word of caution
+
+While nohoist is useful, it does come with drawbacks. The most obvious one is the nohoist modules could be duplicated in multiple locations, denying the benefit of hoisting mentioned above. Therefore, we recommend to keep nohoist scope as small and explicit as possible in your project.
+
+## When will it be available?
+
+[#4979](https://github.com/yarnpkg/yarn/pull/4979) was merged on 1/29/2018. According to the [core team](https://github.com/yarnpkg/yarn/pull/4979#issuecomment-364486016), it should be deployed with **1.4.2** in mid Feb...
```
Comment (Haroenv) [inline _posts/2018-02-15-nohoist.md]: We can release this blog post at the same time as the release I guess

File: _posts/2018-02-15-nohoist.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+## How to use it?
+
+Using nohoist is pretty straightforward. It is driven by the nohoist rules defined in package.json. Starting from 1.4.2, yarn will adopt a new workspaces config format to include the (optional) nohoist setting:
+
+```
+// flow type definition:
+export type WorkspacesConfig = {
+  packages?: Array<string>,
```
Comment (Haroenv) [inline _posts/2018-02-15-nohoist.md]: an example json in both cases would be useful

Comment (connectdotz) [reply _posts/2018-02-15-nohoist.md]: done

File: _posts/2018-02-15-nohoist.md (hunk 1 above)
Comment (connectdotz) [reply _posts/2018-02-15-nohoist.md]: ok, updated the wording.

**Comment by bestander** [comment]
Awesome job, @connectdotz!
Let's merge it and share with everyone

**Pull request merged**